packages/terra-worklist-data-grid/src/subcomponents/ColumnResizeHandle.jsx

```diff
@@ -0,0 +1,106 @@
+
+const propTypes = {
+  /**
+   * Text of the column associated with the divider.
+   */
+  columnText: PropTypes.string.isRequired,
+  /**
+   * Number that specifies the Width of the associated column in pixels.
+   */
+  columnWidth: PropTypes.number.isRequired,
+  /**
+   * Number that specifies the height of the resize handle in pixels.
+   */
+  height: PropTypes.number.isRequired,
+  /**
+   * Number that specifies the minimum column width in pixels.
+   */
+  minimumWidth: PropTypes.number.isRequired,
+  /**
+   * Number that specifies the maximum column width in pixels.
+   */
+  maximumWidth: PropTypes.number.isRequired,
+  /**
+   * Function that is called when onMouseDown event is triggered for the resize handle
+   */
+  onResizeMouseDown: PropTypes.func.isRequired,
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+};
+
+const ColumnResizeHandle = (props) => {
+  const {
+    columnText,
+    columnWidth,
+    height,
+    minimumWidth,
+    maximumWidth,
+    onResizeMouseDown,
+    intl,
+  } = props;
+
+  // State variable to control screen reader visibility
+  const [isActive, setActive] = useState(false);
+
+  // Retrieve current theme from context
+  const theme = useContext(ThemeContext);
+  // Ref variable for native resize handle element
+  const resizeHandle = useRef();
+
+  // Ref callback to obtain the native resize handle element
+  const resizeHandleRef = useCallback((node) => {
+    resizeHandle.current = node;
+  }, []);
+
+  // Mouse down event listener to give focus to resize handler and notify the provider
+  const onMouseDown = (event) => {
+    // Set focus to resize handle DOM element
+    resizeHandle.current.focus();
+
+    // Execute callback function to notify consumer of mouse down event
+    onResizeMouseDown(event);
+
+    // Prevent event bubbling since necessary actions are handled by this component
+    event.stopPropagation();
+    // Prevent default dragging behavior
+    event.preventDefault();
+  };
+
+  return (
+    /* eslint-disable react/forbid-dom-props */
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/interactive-supports-focus
+    <div
+      ref={resizeHandleRef}
+      draggable
+      role="slider"
+      tabIndex={-1}
+      aria-hidden={!isActive}
+      aria-valuemin={minimumWidth}
+      aria-valuenow={columnWidth}
+      aria-valuemax={maximumWidth}
+      aria-label={columnText}
+      aria-valuetext={intl.formatMessage({ id: 'Terra.worklist-data-grid.resizeHandleValueText' }, { columnWidth })}
+      style={{ height: `${height}px` }}
+      onMouseDown={onMouseDown}
+      onFocus={() => setActive(true)}
+      onBlur={() => setActive(false)}
+      className={cx('resize-handle', theme.className)}
+    />
+  );
+};
+
+ColumnResizeHandle.propTypes = propTypes;
```

Comment:
Confused about the component name; is it a `handle` as in a ui element to grab and resize the column or is it a `handler` that takes care of resize events?

Reply:
It is a handle as in a UI element to grab and resize the column.